packages/terra-form-validation/src/FormValidationUtil.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-restricted-globals */
+
+class FormValidationUtil {
+  /* Checks given value's length against maximum. Returns true if value's length is less than the maximum.
+   * @value is the value string, @maximum is a nonnegative whole number. */
+  static isUnderMaxLength(value, maximum) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(typeof maximum === 'number' && isFinite(maximum))
+    || maximum < 0 || !Number.isInteger(maximum)) { return false; }
+    return value.length <= maximum;
+  }
+
+  /* Checks given value's length against minimum. Returns true if value's length is greater than the minimum.
+   * @value is the value string, @minimum is a nonnegative whole number. */
+  static isOverMinLength(value, minimum) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(typeof minimum === 'number' && isFinite(minimum))
+    || minimum < 0 || !Number.isInteger(minimum)) { return false; }
+    return value.length >= minimum;
+  }
+
+  /* Checks given value for extraneous whitespace. return true if there is no whitespace.
+   * @value is the value string */
+  static hasNoWhitespace(value) {
+    if (!(typeof value === 'string' || value instanceof String)) { return false; }
+    return value.trim() === value;
```

Comment:
If this function only intends to check for leading or trailing whitespaces, then it would work. If it should also check for white spaces in between characters, then a value of `white space` would return false. This function could call containsCharacters(value, ' ').

Reply:
Fixed it here https://github.com/cerner/terra-framework/pull/635/commits/415c46ab9a0b74893a686c38430b1d8decd708ad#diff-e0dd485bcf22c1ac58c3d5cfd0103965R27
As per offline discussion instead of using containsCharacters, using a Regex test to see if we have anything that counts as a white space character.